types/mui-image/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for mui-image 1.0
+// Project: https://github.com/benmneb/mui-image
+// Definitions by: benmneb <https://github.com/benmneb>
+//                 Natalia <https://github.com/CodeMeNatalie>
```

Comment:
You have to this person to ACK being added here as owner, as the automation sends notification to onwers handles (so effectively emails owners). Sorry for the inconvience!

Reply:
sorry, please do not skip this one

Reply:
You can give this user credits due as inline comment, but do not put into owners, as autmation will send notifications